README.md

```diff
@@ -1,6 +1,6 @@
 # Contact Picker API
 
-[Link to spec](https://wicg.github.io/contact-api/spec/index.html)
+[Link to spec](https://w3c.github.io/contact-api/spec/index.html)
```

Comment:
All references to the wicg are in the form of repo links. I feel like these should be changed after the repo migration. WDYT?

Reply:
Agree.